assets/stylesheets/govuk-elements/_buttons.scss

```diff
@@ -45,3 +45,8 @@
     background-image: file-url("icons/icon-pointer-2x.png");
   }
 }
+
+.button + .inline-options {
+    padding-top: em(8);
```

Comment:
Not quite sure what you mean.

Here is a screenshot with the padding set to `em(10)` - to my eye the supporting text looks too low. I guess because the underline shifts the visual weight down.

![screenshot from 2015-08-11 12 36 44](https://cloud.githubusercontent.com/assets/117398/9196687/b66d6732-4025-11e5-8081-0ac8c6ab90d3.png)